installer/image_build/files/launch_awx_task.sh

```diff
@@ -6,8 +6,10 @@ if [ `id -u` -ge 10000 ]; then
 fi
 ANSIBLE_REMOTE_TEMP=/tmp ANSIBLE_LOCAL_TEMP=/tmp ansible -i "127.0.0.1," -c local -v -m postgresql_db -U $DATABASE_USER -a "name=$DATABASE_NAME owner=$DATABASE_USER login_user=$DATABASE_USER login_host=$DATABASE_HOST login_password=$DATABASE_PASSWORD" all
 awx-manage migrate --noinput --fake-initial
-echo "from django.contrib.auth.models import User; User.objects.create_superuser('admin', 'root@localhost', 'password')" | awx-manage shell
-awx-manage create_preload_data
+if [ ! -z "$AWX_ADMIN_USER" ]; then
```

Comment:
You might want to check for ```$AWX_ADMIN_PASSWORD``` here too, I know that if users are specifying one they're probably specifying both, but...  Either this or spell out that these variables are required, currently the inventory file says "when set".

```shell
if [ ! -z "$AWX_ADMIN_USER" ]&&[ ! -z "$AWX_ADMIN_PASSWORD" ]; then
```